Updating darwin guide and SHAs (#34860)
project-chip/connectedhomeip: docs/guides/darwin.md

# Matter Development and Testing with/on Apple Devices

### Overview

-   To develop a Matter Application, learn more about how to develop with Matter
    on Apple's platforms [here](https://developer.apple.com/apple-home/matter/)

    -   Additional documentation about the Matter Support API is
        [here](https://developer.apple.com/documentation/mattersupport)

### Filing Issues

-   An issues related to Apple Home, or Development on Apple's Platforms should

>     be reported using this [feedback tool](#providing-feedback-to-apple) as
>     described in this section

Reviewer: ```suggestion
    be reported using Feedback Assistant as
    described [below](#providing-feedback-to-apple)
```

### Enable Developer Mode on your Apple Device

> ##### Note: Developer mode is a great way to get logs from your device as well as enables other useful developer tools

Reviewer: ```suggestion
##### Note: Developer mode is a great way to get logs from your device as well as enable other useful developer tools
```
